tools/apps/src/bin/quiche-server.rs

```diff
@@ -287,7 +287,23 @@ fn main() {
                     hex_dump(&scid)
                 );
 
-                let conn = quiche::accept(&scid, odcid, &mut config).unwrap();
+                let conn = if cfg!(feature = "qlog") && std::env::var_os("QLOGDIR").is_some() {
```

Comment:
Why not just do the `if` and `set_qlog()` after the connection is created like the client does? So we wouldn't suplicate the `accept()` call.

Reply:
so I tried that first but Clippy gets itself tied in knots, if qlog is not enabled, `conn` doesn't need to be `mut`. If it's not ` mut` and qlog is enabled then `set_qlog` will fail. We could disable the cliipy warning in this case?

Reply:
Yeah, silencing should be fine in this case.